internal/common/cache/lru_test.go

```diff
@@ -82,23 +85,39 @@ func TestLRUCacheConcurrentAccess(t *testing.T) {
 		"D": "dank",
 		"E": "ezpz",
 	}
-
-	for k, v := range values {
-		cache.Put(k, v)
+	keys := make([]string, 0, len(values))
```

Comment:
I believe this is no longer used. If yes, please remove this.

Reply:
It is in use: https://github.com/uber-go/cadence-client/blob/master/internal/internal_task_handlers.go#L373

Pinning isn't, but the cache itself is.
(that said, I'm happy to separate this commit from the branch and we can optionally tackle it later - I'll set that up.)

Reply:
removed and pushed a new series of commits btw